LEAF_Request_Portal/templates/reports/LEAF_inbox_combined.tpl

```diff
@@ -1,287 +1,298 @@
             $('#'+data.cellContainerID).attr('tabindex', '0');
             $('#'+data.cellContainerID).attr('aria-label', blob[depID]['records'][data.recordID].title);
             $('#'+data.cellContainerID).html(blob[depID]['records'][data.recordID].title
-                + ' <button id="'+ data.cellContainerID +'_preview" class="buttonNorm">View Request</button>'
-                + '<div id="inboxForm'+ hash +'_' + depID + '_' + data.recordID +'" style="background-color: white; display: none; height: 300px; overflow: scroll"></div>');
+            + ' <button id="'+ data.cellContainerID +'_preview" class="buttonNorm">View Request</button>'
+            + '<div id="inboxForm'+ hash +'_' + depID + '_' + data.recordID +'" style="background-color: white; display: none; height: 300px; overflow: scroll"></div>');
             $('#'+data.cellContainerID + '_preview').on('click', function() {
-                $('#'+data.cellContainerID + '_preview').hide();
-                if($('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).html() == '') {
-                    $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).html('Loading...');
-                    $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).slideDown();
-                    $.ajax({
-                        type: 'GET',
-                        url: site.url + 'ajaxIndex.php?a=printview&recordID=' + data.recordID,
-                        success: function(res) {
-                            $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).html(res);
-                            $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).slideDown();
-                            $('#requestTitle').attr('tabindex', '0');
-                            $('#requestInfo').attr('tabindex', '0');
-                        ariaSubIndicators(1);
-                        }
-                    });
-                }
-            });
-        }},
-        {name: 'Action', indicatorID: 'action', editable: false, sortable: false, callback: function(data, blob) {
-            var depDescription = 'Take Action';
+            $('#'+data.cellContainerID + '_preview').hide();
+            if($('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).html() == '') {
+                $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).html('Loading...');
+                $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).slideDown();
+                $.ajax({
+                    type: 'GET',
+                    url: site.url + 'ajaxIndex.php?a=printview&recordID=' + data.recordID,
+                    success: function(res) {
+                     $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).html(res);
+                     $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).slideDown();
+                     $('#requestTitle').attr('tabindex', '0');
+                     $('#requestInfo').attr('tabindex', '0');
+                     ariaSubIndicators(1);
+                    }
+                })
+            }
```

Comment:
Is there a reason these brackets are all stacked vertically?  Not critical, but would be hard to read upon an initial view of the code.